src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -413,13 +451,57 @@ public SolutionState RemoveProjects(ArrayBuilder<ProjectId> projectIds)
         foreach (var projectId in projectIds)
             newDependencyGraph = newDependencyGraph.WithProjectRemoved(projectId);
 
+        var languageCountDeltas = new TemporaryArray<(string language, int count)>();
+        foreach (var projectId in projectIds)
+        {
+            AddLanguageCount(ref languageCountDeltas, _projectIdToProjectStateMap[projectId].Language, amount: -1);
```

Comment:
ah, i think this should be called 'deltaAmount'

Reply:
The method is named "Add-", so amount seemed like the right name for adding some quantity.

Reply:
deltaAmount sounds like second derivative :)